prow/scripts/cluster-integration/kyma-gke-end-to-end-test.sh

```diff
@@ -48,32 +48,40 @@ removeCluster() {
     #Turn off exit-on-error so that next step is executed even if previous one fails.
     set +e
 
-    EXIT_STATUS=$?
-
-    shout "Fetching OLD_TIMESTAMP from cluster to be deleted"
-	readonly OLD_TIMESTAMP=$(gcloud container clusters describe "${CLUSTER_NAME}" --zone="${GCLOUD_COMPUTE_ZONE}" --project="${GCLOUD_PROJECT_NAME}"  --format=json | jq --raw-output '.createTime' | cut -f1 -d"T" | sed "s/-//g")
-
     shout "Deprovision cluster: \"${CLUSTER_NAME}\""
     date
 
-    #save disk names while the cluster still exists to remove them later
-    DISKS=$(kubectl get pvc --all-namespaces -o jsonpath="{.items[*].spec.volumeName}" | xargs -n1 echo)
-    export DISKS
+    EXIT_STATUS=$?
```

Comment:
Moving this below the date command will catch save the error of the date command in the exit status. I don't think that is what you wanted to achieve here.